atomicapp/constants.py

```diff
@@ -52,10 +53,10 @@
 DEFAULT_PROVIDER = "kubernetes"
 DEFAULT_CONTAINER_NAME = "atomic"
 DEFAULT_NAMESPACE = "default"
-DEFAULT_ANSWERS = {
+DEFAULT_ANSWERS_FORMAT = {
     "general": {
-        "provider": DEFAULT_PROVIDER,
-        "namespace": DEFAULT_NAMESPACE
+        "provider": None,
+        "namespace": "default"
```

Comment:
If I specify None, then it's much easier to pick it up via determine if a provider has been passed or not. If we have to revert back to using DEFAULT_PROVIDER, I would have to determine and check to see if something is "Kubernetes" or not which would be bad practice.  I don't know how else I should explain, but passing in a template would be more ideal than relying on a default preference.